### PR TITLE
Rename project to adapters.

### PR DESCRIPTION
**Pull request by mjtamlyn**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Django serial forms's documentation!
+Welcome to Django adapters's documentation!
 ===============================================
```
Comment (MoritzS) [inline docs/source/index.rst]: Too much `===` for the heading

File: setup.py [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 
 setup(
-    name='django-serial-forms',
-    py_modules=['serial_forms'],
+    name='django-adapters',
+    py_modules=['adapters'],
     version='0.1',
-    description='Constructible, introspectable serializers and forms',
+    description='Constructible, introspectable adaption for serialization and forms',
```
Comment (MoritzS) [inline setup.py]: For me "adaption" sounds quite different than what "adapter" would mean, maybe change it to "adapters"?

File: docs/source/index.rst (hunk 1 above)
Comment (mjtamlyn) [inline docs/source/index.rst]: Done :)

**Comment by MoritzS** [comment]
Line 1 of `docs/source/index.rst` still says serial forms.

**Comment by MoritzS** [comment]
Line 1 of `docs/source/index.rst` still says serial forms.

**Comment by mjtamlyn** [comment]
That comment has been deleted in #20 anyway.

**Pull request merged**